lib/ansible/modules/remote_management/ucs/ucs_uuid_pool.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ucs_uuid_pool
+short_description: Configures server UUID pools on Cisco UCS Manager
+description:
+- Configures server UUID pools and UUID blocks on Cisco UCS Manager.
+- Examples can be used with the UCS Platform Emulator U(https://communities.cisco.com/ucspe).
+extends_documentation_fragment: ucs
+options:
+  state:
+    description:
+    - If C(present), will verify UUID pool is present and will create if needed.
+    - If C(absent), will verify UUID pool is absent and will delete if needed.
+    choices: [present, absent]
+    default: present
+  name:
+    description:
+    - The name of the UUID pool.
+    - This name can be between 1 and 32 alphanumeric characters.
+    - "You cannot use spaces or any special characters other than - (hyphen), \"_\" (underscore), : (colon), and . (period)."
+    - You cannot change this name after the UUID pool is created.
+    required: yes
+  description:
+    description:
+    - "The user-defined description of the UUID pool."
+    - Enter up to 256 characters.
+    - "You can use any characters or spaces except the following:"
```

Comment:
This line does not need quotes. (nitpick)

Reply:
With the colon, I had to quote for ansible-doc to accept the line.

Reply:
Oh right, trailing colon.